Playbooks/playbook-Prisma_Cloud_Remediation_-_AWS_Inactive_users_for_more_than_30_days.yml

```diff
@@ -0,0 +1,526 @@
+      version: -1
+      name: Is AWS - IAM integration enabled?
+      description: Check to ensure that IAM integration is configured
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "18"
+      "yes":
+      - "15"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: AWS - IAM
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.state
+                      iscontext: true
+                    right:
+                      value:
+                        simple: active
+                accessor: brand
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 152.5,
+          "y": 1070
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "7":
+    id: "7"
+    taskid: ffcd7bb2-29a0-440c-8856-c6afe386fae9
+    type: title
+    task:
+      id: ffcd7bb2-29a0-440c-8856-c6afe386fae9
+      version: -1
+      name: Done
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 2205
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "14":
+    id: "14"
+    taskid: 4dbc18f2-3905-4773-81ad-6fc103bc2705
+    type: regular
+    task:
+      id: 4dbc18f2-3905-4773-81ad-6fc103bc2705
+      version: -1
+      name: Print user credential report
+      description: Outputs user's credential report which contains access keys attributes
+        and  user login profile
+      scriptName: Print
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      value:
+        complex:
+          root: incident
+          accessor: labels.resource
+          transformers:
+          - operator: ParseJSON
+          - operator: getField
+            args:
+              field:
+                value:
+                  simple: data
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 410
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "15":
+    id: "15"
+    taskid: a171957c-457e-4879-81a8-d964e4755796
+    type: regular
+    task:
+      id: a171957c-457e-4879-81a8-d964e4755796
+      version: -1
+      name: Listing access keys
```

Comment:
Tasks names should be *_List access keys, Disable access keys_* instead of _Listing access keys, Disabling Access keys_.
(Relevant for all task names)